.github/workflows/tests.yml

```diff
@@ -15,8 +15,11 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+          registry-url: 'https://registry.npmjs.org/'
```

Comment:
why do we need this?

Reply:
Cause @wixc3/isomorphic-worker lives there, and I got these errors without this change:
https://github.com/wixplosives/engine/actions/runs/5110105664/jobs/9185624849#step:6:13